stdlib/2and3/ctypes/__init__.pyi

```diff
@@ -0,0 +1,257 @@
+# Stubs for ctypes
+
+from typing import (
+    Any, Callable, ClassVar, Iterable, List, Mapping, Optional, Sequence, Sized, Tuple, Type,
+    Generic, TypeVar, overload,
+)
+from typing import Union as UnionT
```

Comment:
You should name this `_UnionT` so it doesn't get re-exported. (With the current definition, mypy would let users write `ctypes.UnionT`.)